<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIHerramienta</class>
 <widget class="QMainWindow" name="UIHerramienta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DPLL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/iconos/iconoVentana.png</normaloff>:/iconos/iconoVentana.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-image: url(:/iconos/gradient7.png);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowserInfo">
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/iconos/info_dpll.png&quot; /&gt;&lt;img src=&quot;:/iconos/info_eliminacionUnitaria.png&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/iconos/info_eliminacionPura.png&quot; /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/iconos/info_ejemploDpll.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelClausula">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Ingrese su clausula aquí, separando los literales entre comas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineClausula"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="botonNegacion">
        <property name="text">
         <string>~</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="botonCargar">
        <property name="text">
         <string>Cargar clausula</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonBorrar">
        <property name="text">
         <string>Borrar último</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonVaciar">
        <property name="text">
         <string>Borrar todo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="espacio0">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color: transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelConjunto">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QMainWindow
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Conjunto:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="espacio1">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonResolver">
      <property name="text">
       <string>Ver resolución</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonValuaciones">
      <property name="text">
       <string>Ver valuaciones</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>botonResolver</zorder>
   <zorder>labelConjunto</zorder>
   <zorder>textBrowserInfo</zorder>
   <zorder>textBrowser</zorder>
   <zorder>espacio0</zorder>
   <zorder>espacio1</zorder>
   <zorder>botonValuaciones</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir_Archivo"/>
    <addaction name="actionGuardar_Archivo"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionQue_es_DPLL"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAtras"/>
  </widget>
  <widget class="QToolBar" name="toolBarInfo">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAtrasInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir_Archivo">
   <property name="icon">
    <iconset>
     <normaloff>icono_abrir.png</normaloff>icono_abrir.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Archivo </string>
   </property>
  </action>
  <action name="actionGuardar_Archivo">
   <property name="icon">
    <iconset>
     <normaloff>icono_guardar.png</normaloff>icono_guardar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar Archivo</string>
   </property>
  </action>
  <action name="actionQue_es_DPLL">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/configuration12.png</normaloff>:/iconos/configuration12.png</iconset>
   </property>
   <property name="text">
    <string>¿Qué es Davis Putnam Logemann Loveland?</string>
   </property>
  </action>
  <action name="actionAtras">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/back28.png</normaloff>:/iconos/back28.png</iconset>
   </property>
   <property name="text">
    <string>Atras</string>
   </property>
  </action>
  <action name="actionAtrasInfo">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/back28.png</normaloff>:/iconos/back28.png</iconset>
   </property>
   <property name="text">
    <string>AtrasInfo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineClausula</sender>
   <signal>returnPressed()</signal>
   <receiver>botonCargar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
